README.md

```diff
@@ -21,6 +21,16 @@ NMState is aimed to satisfy enterprise needs to manage host networking through
 a northbound declarative API and multi provider support on the southbound.
 NetworkManager acts as the main (and currently the only) provider supported.
 
+## Contact
+nmstate uses the Network [Manager mailing
+list](https://mail.gnome.org/mailman/listinfo/networkmanager-list)
+([Archives](https://mail.gnome.org/archives/networkmanager-list/)) for
+discussions. Emails about nmstate should be tagged with `[nmstate]` in the
+subject header to ease filtering.
+
+Please also ([Jira](https://nmstate.atlassian.net)) for planning the current
```

Comment:
How about: Jira is used for managing the development tasks, covering sprint planning, bugs and progress.

Reply:
I reworded it, hopefully this is what you meant. I did not mention bugs, yet, since we would need to write something about Github issues then as well.